presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -379,6 +379,14 @@ Accessors
 
     Returns the cardinality of the collection of interior rings of a polygon.
 
+.. function:: line_interpolate_point(LineString, double) -> double
+
+    Returns a Point interpolated along a LineString at the fraction given.
```

Comment:
Add a note about the double argument having to be in the `[0, 1]` range and any requirements on LineString.